File: fizzbuzz.php
<?php

for ($i=1; $i<=100; $i++){
    if (($i % 15) == 0){
        print "FizzBuzz";
        print "<br>";
    }
    elseif (($i % 3) == 0){
        print "Fizz";
        print "<br>";
    }
    elseif (($i % 5) == 0){
        print "Buzz";
        print "<br>";
    }
    else{
        print $i;
        print "<br>";
    }
}

?>

File: fizzbuzz.rb
range = 1..100
range.each do |num|
    if num % 15 == 0
        p "FizzBuzz"
    elsif num % 3 == 0
        p "Fizz"
    elsif num % 5 == 0
        p "Buzz"
    else
        p num
    end
end